libexec/scoop-checkup.ps1

```diff
@@ -13,6 +13,30 @@ $issues += !(check_windows_defender $true)
 $issues += !(check_main_bucket)
 $issues += !(check_long_paths)
 
+if (!(Test-HelperInstalled -Helper 7zip)) {
+    error "'7-Zip' is not installed! It's required for unpacking most programs. Please Run 'scoop install 7zip' or 'scoop install 7zip-zstd'."
+    $issues++
+}
+
+if (!(Test-HelperInstalled -Helper Lessmsi)) {
```

Comment:
Less msi is optional dependency. Should not be reported.

Reply:
```powershell
if (!(Test-HelperInstalled -Helper Lessmsi)) {
    warn "'lessmsi' is not installed. (Optional) This may resolve problems MSI packages. You may run 'scoop install lessmsi'."
}
```

Reply:
Again. Same as for aria2.

When i do not want to use lessmsi i have no option how to disable this check.